src/core/src/op/eyelike.cpp

```diff
@@ -0,0 +1,122 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "ngraph/op/eyelike.hpp"
+
+#include <ngraph/validation_util.hpp>
```

Comment:
I think you can remove this header
```suggestion
```

Reply:
Unfortunately I cannot because I use `get_constant+from_source()`